manifests/init.pp

```diff
@@ -31,6 +31,13 @@
 #                               type:Stdlib::Absolutepath
 #
 # $localzonepath::              File holding local zones like RFC1912 or RFC1918 files.
+#                               The special value 'unmanaged' can be used if one plans
+#                               to create custom RFC1912/RFC1918 zones via ::dns,
+#                               where the inclusion of package-shipped zone files is
+#                               not desired.
+#                               type:Optional[Variant[Enum['unmanaged'],Stdlib::Absolutepath]]
```

Comment:
These types are no longer needed and were removed in https://github.com/theforeman/puppet-dns/commit/87df08cd842a5465caa404d1dabe1213e8218020 (which is also causing a merge conflict.